views/shared/paginator.html

```diff
@@ -0,0 +1,32 @@
+{{ if .paginator.HasPages }}
+  <ul class="pagination justify-content-center">
+    {{ if .paginator.HasPrev }}
+      <li class="page-item">
```

Comment:
These class names do not follow BEM naming conventions:

```suggestion
      <li class="pagination__page-item">
```

Reply:
I was using bootstrap pagination component, now I make them to BEM strutured https://github.com/junan/google-scraper/pull/34/commits/8cd54f1603bbae64255ad29d8776cf82251c2b2e